specification/service-search-api.yaml

```diff
@@ -18,4 +157,483 @@ servers:
     description: Integration test environment.
   - url: 'https://api.service.nhs.uk/service-search-api'
     description: Production environment.
-paths: {}
+paths:
+
+  "":
+  
+    # --------------------------------------------------------------------------------
+    # GET /
+    # --------------------------------------------------------------------------------
+  
+    get:
+      operationId: root-GET
+      description: Information about services provided by NHS organisations listed in the service search on the NHS website.
+      parameters:
+        - name: api-version
+          description: The API version number can be supplied as a query parameter.
+          in: query
+          schema:
+            type: number
+            default: 1
+            enum: [1]
+        - name: search
+          in: query
+          description: The text to search for. All searchable fields are searched by default unless searchFields is specified. When searching searchable fields, the search text itself is tokenized, so multiple terms can be separated by white space (for example search=hello world). To match any term, use * (this can be useful for boolean filter queries). Omitting this parameter has the same effect as setting it to *.
```

Comment:
Is this actually a default * then?

Reply:
No, this is a constant.  To access this version of the api you need to do `api-version=1`.  Swagger doesn't support JSON schema `const` and this is the work around. 

```
enum: [1]
default: 1
```

Reply:
Line on 181. Description basically says it is a default value but we have no default field

Reply:
Yeah, I've done that now.  Not needed but clarifies it.